zeppelin-interpreter/src/main/java/org/apache/zeppelin/conf/ZeppelinConfiguration.java

```diff
@@ -905,6 +910,10 @@ public boolean isPrometheusMetricEnabled() {
     return getBoolean(ConfVars.ZEPPELIN_METRIC_ENABLE_PROMETHEUS);
   }
 
+  public DEFAULT_UI getDefaultUi() {
+    return DEFAULT_UI.valueOf(getString(ConfVars.ZEPPELIN_DEFAULT_UI).toUpperCase());
```

Comment:
What happens if I pass neither `new` nor `classic` as a configuration value at this point?

Reply:
If neither new nor classic is provided as a configuration value, the valueOf() method will throw an IllegalArgumentException during server startup.
I based this behavior on the getRunMode() method, which similarly doesn't handle exceptions for invalid input.

Should we add more graceful error handling here?

Reply:
@jongyoul If the value is empty, `getString()` will return default value (`new`) and if an invalid value (neither `new` nor `classic`) is provided, it will throw an exception.

Do you think it is not appropriate to provide a default value for an empty value in this configuration?

Reply:
In my opinion, the exception and the termination of the start in the event of incorrect configuration is acceptable.